### PR TITLE
Support S-Expression syntax in stubs

### PR DESCRIPTION
**Pull request by ellnix**
Languages like lisp, clojure, or webassembly have [S-Expression](https://en.wikipedia.org/wiki/S-expression) syntax which cannot be generated from the current way we store stubs.

### CODE REVIEW

**Review by Andriamanitra — CHANGES_REQUESTED** [review]
I think the preprocessors and the 'ad-hoc' commands they introduce are a messy solution, but it works well for now and I don't have any better ideas.

The only thing keeping this from getting merged right now is that the `s-expression` preprocessor should be renamed to something more accurate (as discussed on Discord the other day).

I'd be fine with any of these suggestions (or something similar):
* `lisp-like`
* `batch-read-with-lexical-scope`
* `read-batch-with-explicit-scope`

**Comment by ellnix** [comment]
>     * `read-batch-with-explicit-scope`

This is probably my favorite, since it hints that this preprocessor is why there is a read_batch template, but we can do a poll.

File: src/stub/preprocessor/lisp_like.rs [hunk 1]
```diff
@@ -1,6 +1,15 @@
 use super::Renderable;
 use crate::stub::{Cmd, Stub};
 
+/// Edit Stub to allow for rendering lisp-like syntax
```
Comment (daxida) [inline src/stub/preprocessor/lisp_like.rs]: Final dot after "syntax" to be consistent :D

**Review by daxida — APPROVED** [review]
Everything seems fine to me :+1:

File: src/stub/preprocessor/lisp_like.rs (hunk 1 above)
Comment (ellnix) [reply src/stub/preprocessor/lisp_like.rs]: You got me xD

**Review by Andriamanitra — APPROVED** [review]

**Pull request merged**